plugins/org.jkiss.dbeaver.ext.cubrid.ui/src/org/jkiss/dbeaver/ext/cubrid/ui/editors/CubridSQLPlanViewProvider.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * DBeaver - Universal Database Manager
+ * Copyright (C) 2010-2024 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jkiss.dbeaver.ext.cubrid.ui.editors;
+
+import org.eclipse.jface.viewers.Viewer;
+import org.jkiss.dbeaver.DBException;
+import org.jkiss.dbeaver.Log;
+import org.jkiss.dbeaver.model.exec.plan.DBCPlan;
+import org.jkiss.dbeaver.model.sql.SQLQuery;
+import org.jkiss.dbeaver.ui.editors.sql.plan.simple.SQLPlanViewProviderSimple;
+
+public class CubridSQLPlanViewProvider extends SQLPlanViewProviderSimple
+{
+    private static final Log log = Log.getLog(CubridSQLPlanViewProvider.class);
+
+    @Override
+    public void visualizeQueryPlan(Viewer viewer, SQLQuery query, DBCPlan plan) {
+        try {
+            query.setText(plan.getPlanQueryString());
+        } catch (DBException e) {
+            log.debug("Cubrid: could not set query text", e);
+        }
```

Comment:
You know that the plan is always an instance of `CubridPlanAnalyser`.
```suggestion
        query.setText(((CubridPlanAnalyser) plan).getPlanQueryString());
```